plutus-use-cases/test/Spec/Vesting.hs

```diff
@@ -4,6 +4,7 @@
 {-# LANGUAGE RecordWildCards     #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TemplateHaskell     #-}
+{-# OPTIONS_GHC -fno-warn-incomplete-uni-patterns -fno-warn-unused-do-bind #-}
```

Comment:
I assume this is because you're using a different version of the compiler to everyone else?

Reply:
No, this is triggered by GHC 8.2.2

Reply:
Huh, that *is* weird, I don't know how that got past the CI?